Routs/tasksRouts.js

```diff
@@ -0,0 +1,120 @@
+module.exports = function(app, db){
+    const {v4: uuidv4} = require('uuid')
+
+    function GenerateUUID()
+    {
+        return uuidv4()
+    }
+
+    app.get('/api/tasks/:id', function(req, res){
+        var data = db.ReadData()
+        var JSON_obj_file = JSON.parse(data)
+    
+        var task_id = req.params.id
+
+        if (task_id in JSON_obj_file)
+        {
+            res.send(JSON_obj_file[task_id])
+        }
+        else
+        {
+            console.log("ID isn't exist")
+            res.status(404).json({"errorMessage":"ID isn't exist"})
+        }
+    })
+    
+    app.get('/api/tasks', function(req, res){
+        var data = db.ReadData()
+        var JSON_obj_file = JSON.parse(data)
+    
+        var tasks_Array = []
+        Object.values(JSON_obj_file).map(task => tasks_Array.push(task))
+        
+        res.send(tasks_Array)
+    })
+    
+    app.get('*', function(req, res){
+        res.status(404).send('Looks like you have lost the path ;)');
+    })
+    
+    app.post('/api/task', function(req, res){
+        var data = db.ReadData()
+        var JSON_obj_file = JSON.parse(data)
+        
+        var JSON_obj_task = req.body
+    
+        JSON_obj_task.createTime = new Date().toLocaleString()
```

Comment:
great!
I don't know if it will work but try without the toLocaleString and see if it will work when you save it to the json